.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "Xamarin.Auth"]
```

Comment:
Does it make sense to keep this as a submodule, or should it just be moved to a NuGet dependency?